opengrok-indexer/src/main/java/org/opengrok/indexer/web/Util.java

```diff
@@ -1369,17 +1390,18 @@ public Void apply(Integer page) {
                         // append request parameters
                         if (request != null && request.getQueryString() != null) {
                             String query = request.getQueryString();
-                            query = query.replaceFirst("(\\?|&amp;|&|)n=\\d+", "");
-                            query = query.replaceFirst("(\\?|&amp;|&|)start=\\d+", "");
-                            query = query.replaceFirst("^(\\?|&amp;|&)", "");
+                            query = query.replaceFirst(RE_Q_E_A_A_COUNT_EQ_VAL, "");
+                            query = query.replaceFirst(RE_Q_E_A_A_START_EQ_VAL, "");
+                            query = query.replaceFirst(RE_A_ANCHOR_Q_E_A_A, "");
```

Comment:
these constants are a little bit cryptic on the first sight but they actually make sense.